agent/spec/lib/kontena/workers/node_stats_worker_spec.rb

```diff
@@ -138,12 +129,13 @@
   describe '#publish_node_stats' do
 
     before(:each) do
+      subject.instance_variable_set('@node', node)
```

Comment:
It's set in the `start` -> `observe` block from the node info sent by the server... could refactor that out and call that from `before`.

#2591 also removes the need for the `@node` here entirely.